packaging/keybase.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
```

Comment:
Can you make this `#!/bin/bash`.  I believe @cjb hit a bug in which he tried to run the script with `sh` and failed.